test/fx/test_gradual_type.py

```diff
@@ -7,6 +7,11 @@
 from torch.fx.experimental.rewriter import RewritingTracer
 from torch.fx import GraphModule
 
+def conv3x3(in_planes, out_planes, stride=1, groups=1, dilation=1):
+    """3x3 convolution with padding"""
+    return torch.nn.Conv2d(in_planes, out_planes, kernel_size=3, stride=stride,
+                           padding=dilation, groups=groups, bias=False, dilation=dilation)
```

Comment:
is `padding=dilation` intended?

Reply:
yeah, I copied this function from torchvision_models.py and it has padding = dilation for some reason.